tensorflow_addons/options.py

```diff
@@ -37,13 +36,34 @@
 import ...
 import ...
 
-tfa.options.TF_ADDONS_PY_OPS = True
+tfa.options.disable_custom_kernel()
 ```
 """
 
 
 def warn_fallback(op_name):
     warning_msg = FALLBACK_WARNING_TEMPLATE.format(traceback.format_exc(), op_name)
     warnings.warn(warning_msg, RuntimeWarning)
-    global TF_ADDONS_PY_OPS
-    TF_ADDONS_PY_OPS = True
+    global _TF_ADDONS_PY_OPS
+    _TF_ADDONS_PY_OPS = True
+
+
+def enable_custom_kernel():
+    """Enable using custom CUDA/C++ Kernel instead of pure python kernel. Use this instead of
+    directly accessing the global variable."""
+    global _TF_ADDONS_PY_OPS
+    _TF_ADDONS_PY_OPS = False
+    pass
+
+
+def disable_custom_kernel():
+    """Disable using custom CUDA/C++ Kernel instead of pure python kernel. Use this instead of
+    directly accessing the global variable."""
+    global _TF_ADDONS_PY_OPS
+    _TF_ADDONS_PY_OPS = True
+    pass
+
+
+def py_enabled():
```

Comment:
Hmm, there should be a suitable name here as our API focus on whether to enable custom kernel?

Reply:
custom_enabled()?

Reply:
How about `custom_kernel_enabled`?

Reply:
Because if python is preferred that means custom is disabled, it will be convenient to the reader as well.

Reply:
Yeah sure, ```custom_kernel_disabled```! (because it TF_ADDONS_PY_OPS is true it means custom is disabled)

Reply:
I prefer it in the `is_` style like:
https://www.tensorflow.org/api_docs/python/tf/math/is_inf
https://www.tensorflow.org/api_docs/python/tf/math/is_nan

Reply:
is_custom_disabled()?

Reply:
how about `is_custom_kernel_disabled`